src/api.js

```diff
@@ -19,6 +19,15 @@ export function getReviewByID(reviewID) {
             return res.data
         })
 }
+
+export function voteReview(reviewID, voteRequest) {
```

Comment:
Well done extracting this out into a separate api call to deal with the patch.

I think that the function name `voteReview` could be more descriptive of what you want the function to do, such as `patchReviewVote` or something along those lines.